akka-http-core/src/test/scala/akka/http/impl/engine/client/ClientConfigurationSpec.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package akka.http.impl.engine.client
+
+import akka.stream.testkit.AkkaSpec
+
+class ClientConfigurationSpec extends AkkaSpec {
+  "Reference configurations" should {
+    for {
+      (first, second) ← "akka.http.client" -> "akka.http.host-connection-pool.client" ::
+        "akka.http.client.parsing" -> "akka.http.host-connection-pool.client.parsing" ::
+        "akka.http.server.parsing" -> "akka.http.client.parsing" ::
+        Nil
+    } s"be consistent for: `$first` and `$second`" in {
+      configShouldBeEqual("akka.http.client.parsing", "akka.http.host-connection-pool.client.parsing")
+    }
```

Comment:
currently complete equality, if we change things only for the server for example this can be changed to check if keys are the same